src/main/java/com/wenqi/bootwebdemo/controller/PersonController.java

```diff
@@ -0,0 +1,42 @@
+package com.wenqi.bootwebdemo.controller;
+
+import com.wenqi.bootwebdemo.exception.PersonNotFoundException;
+import com.wenqi.bootwebdemo.model.Person;
+import com.wenqi.bootwebdemo.model.PersonResponseDTO;
+import com.wenqi.bootwebdemo.service.PersonService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class PersonController {
+
+    @Autowired
+    PersonService personService;
+
+    @GetMapping(path = "/persons", produces = {"application/json"})
+    public List<Person> getPersonList() {
+        List<Person> personList = personService.getAllPersons();
+        return personList;
+    }
+
+    @GetMapping("/person/{aid}")
+    public String getPersonById(@PathVariable("aid") int aid) {
+        String speech = null;
+        try {
+            speech = personService.getPersonById(aid);
+        } catch (PersonNotFoundException e) {
+            return "Oops! No person found";
```

Comment:
exceptions better be logged

Reply:
Log as which level is more suitable in this scenario, INFO or DEBUG or ERROR?

Reply:
> exceptions better be logged

Log as which level is more suitable in this scenario, INFO or DEBUG or ERROR?

Reply:
> exceptions better be logged

This exception is threw from service and catched by controller, where do we put the log or both?

Reply:
> > exceptions better be logged
> 
> Log as which level is more suitable in this scenario, INFO or DEBUG or ERROR?

Warn for non core and pre-assumed possible exceptions.